docs/dev_guide.rst

```diff
@@ -23,6 +23,18 @@ and then use ``fit`` or ``transform`` methods to apply the algorithm to data (ge
 Installation for Development
 ----------------------------
 
+First, you need to clone the NiMARE repository on github.
+
+.. code-block:: bash
+
+  git clone https://github.com/neurostuff/NiMARE.git
```

Comment:
I think creating a fork and cloning that is a necessary step for folks who want to install NiMARE for development.